.github/workflows/release.yml

```diff
@@ -44,7 +44,7 @@ jobs:
             target: x86_64-unknown-linux-gnu
             binary: x86_64-manylinux2014-rocm5.7
             container: rocm/dev-ubuntu-22.04:5.7-complete
```

Comment:
this doesn't seems to be an actual manylinux build - could you confirm?

Reply:
Probably not I guess? Pretty sure there's no container for ROCm for that, I can probably just rename the binary. The resulting binary is fine, it doesn't link against anything special:
```
~$ ldd /tmp/tabby_x86_64-manylinux2014-rocm5.7
	linux-vdso.so.1 (0x00007ffcdaac1000)
	libstdc++.so.6 => /lib/x86_64-linux-gnu/libstdc++.so.6 (0x00007f5757e00000)
	libamdhip64.so.5 => /opt/rocm-5.7.0/lib/libamdhip64.so.5 (0x00007f5756200000)
	librocblas.so.3 => /opt/rocm-5.7.0/lib/librocblas.so.3 (0x00007f5736800000)
	libhipblas.so.1 => /opt/rocm-5.7.0/lib/libhipblas.so.1 (0x00007f575bee1000)
	libssl.so.3 => /lib/x86_64-linux-gnu/libssl.so.3 (0x00007f575be3d000)
	libcrypto.so.3 => /lib/x86_64-linux-gnu/libcrypto.so.3 (0x00007f5736200000)
	libgcc_s.so.1 => /lib/x86_64-linux-gnu/libgcc_s.so.1 (0x00007f575be1b000)
	libm.so.6 => /lib/x86_64-linux-gnu/libm.so.6 (0x00007f5758119000)
	libc.so.6 => /lib/x86_64-linux-gnu/libc.so.6 (0x00007f5735e00000)
	/lib64/ld-linux-x86-64.so.2 (0x00007f575bf78000)
	libamd_comgr.so.2 => /opt/rocm-5.7.0/lib/libamd_comgr.so.2 (0x00007f572d400000)
	libhsa-runtime64.so.1 => /opt/rocm-5.7.0/lib/libhsa-runtime64.so.1 (0x00007f572d000000)
	libnuma.so.1 => /lib/x86_64-linux-gnu/libnuma.so.1 (0x00007f575be0c000)
	librocsolver.so.0 => /opt/rocm-5.7.0/lib/librocsolver.so.0 (0x00007f56cf200000)
	libz.so.1 => /lib/x86_64-linux-gnu/libz.so.1 (0x00007f575bdf0000)
	libtinfo.so.6 => /lib/x86_64-linux-gnu/libtinfo.so.6 (0x00007f575bdbe000)
	libelf.so.1 => /lib/x86_64-linux-gnu/libelf.so.1 (0x00007f575bda0000)
	libdrm.so.2 => /opt/amdgpu/lib/x86_64-linux-gnu/libdrm.so.2 (0x00007f57580ff000)
	libdrm_amdgpu.so.1 => /opt/amdgpu/lib/x86_64-linux-gnu/libdrm_amdgpu.so.1 (0x00007f57580f1000)
	librocsparse.so.0 => /opt/rocm-5.7.0/lib/librocsparse.so.0 (0x00007f568d400000)
```

Reply:
In that case - I would suggest just rely on docker build for rocm atm, non manylinux build won't be very useful (as it's specific to certain distributions)

Reply:
It doesn't rely on anything distro specific, so it should be fine. Almost no one uses manylinux outside of python and there are still rarely problems. Seems like you have a strong python background.

Trying some rando manylinux image from pytorch though, maybe it'll work (if GH decides to actually run that build...)

Reply:
Using hipblas on top of manylinux, this should be fixed now